Switch/Switch.module.less

```diff
@@ -36,6 +40,11 @@
 			.icon {
 				-webkit-transition: left 0.2s, color 0.2s;
 				transition: left 0.2s, color 0.2s;
```

Comment:
Instead of below mixin, we could just use the current `transition` rule with `right` property added.
```
transition: left 0.2s, right 0.2s, color 0.2s;
```